tokens/tokens-eth.json

```diff
@@ -10603,7 +10628,7 @@
 			"gitter": "",
 			"instagram": "",
 			"linkedin": "",
-			"reddit": "https://www.reddit.com/r/requestnetwork",
```

Comment:
@rblxchris did you mean to update RCT token's info?

Reply:
I didn't mean too I made a mistake